components/playlist/features.cc

```diff
@@ -9,6 +9,10 @@
 
 namespace playlist::features {
 
+#ifndef NDEBUG
```

Comment:
1. please avoid NDEBUG, use DCHECK_IS_ENABLED() instead. Usually you don't want to distinguish debug/release builds, but instead "debuggable" or not builds. DCHECKs can be enabled in "release" configuration which makes DCHECK_IS_ENABLED() a better choice for such ifdefs.
2. put only the feature state under the ifdef block, not the whole feature declaration.

Reply:
Cool. You were to say DCHECK_IS_ON(), werent you? Replaced it.

Reply:
Oh, it seems enabling this by default when DCHECK_IS_ON() makes tests flaky. I'll try this later on.